deployments/helm/hephaestus/templates/controller/cronjob.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "hephaestus.manager.name" . }}-cronjob
+  labels: {{- include "hephaestus.controller.labels.standard" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.controller.ibGarbageCollector.interval | quote }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            checksum/config: {{ include (print $.Template.BasePath "/controller/secret.yaml") . | sha256sum }}
+            {{- with .Values.controller.podAnnotations }}
+            {{- toYaml . | trimSuffix "\n" | nindent 12 }}
+            {{- end }}
+            {{- with .Values.podAnnotations }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          labels:
+              {{- include "hephaestus.controller.labels.matchLabels" . | nindent 12 }}
+              {{- with .Values.controller.podLabels }}
+              {{- toYaml . | trimSuffix "\n" | nindent 12 }}
+              {{- end }}
+              {{- with .Values.podLabels }}
+              {{- toYaml . | nindent 12 }}
+              {{- end }}
+        spec:
+          securityContext:
+            {{- toYaml .Values.controller.podSecurityContext | nindent 12 }}
+          restartPolicy: OnFailure
```

Comment:
Generally I'd rather see jobs have a `restartPolicy: Never` and and an explicit, configurable backoffLimit
https://kubernetes.io/docs/concepts/workloads/controllers/job/#handling-pod-and-container-failures

Otherwise, I think the job will stay failing and restarting indefinitely